app/api/server/v1/voip/rooms.ts

```diff
@@ -7,7 +7,21 @@ import { API } from '../../api';
 import { VoipRoom, LivechatVisitors, Users } from '../../../../models/server/raw';
 import { LivechatVoip } from '../../../../../server/sdk';
 import { ILivechatAgent } from '../../../../../definition/ILivechatAgent';
+import { hasPermission } from '../../../../authorization/server';
+import { typedJsonParse } from '../../../../../lib/typedJSONParse';
 
+type DateParam = { start?: string; end?: string };
```

Comment:
Wait, are returning a empty object at the end... even if date is a DateParam instead a string?